Sortable.js

```diff
@@ -736,9 +736,10 @@
 
 
 				if ((el.children.length === 0) || (el.children[0] === ghostEl) ||
-					(el === evt.target) && (_ghostIsLast(el, evt))
+					(el === evt.target) && (target = _ghostIsLast(el, evt))
```

Comment:
> I don't understand, why do first assign and then delete the value?

The point of #1106 was to conditionally set target in first `if` block. But the second `elseif` relies on it so it needs to be set before that point is hit. Unassigning is a compromise that follows the logic of #1106 while eliminating the regression that was caused by it. 

IMO it is a valid regression no matter if we're talking nested or not.

This is the Drupal.org issue that brought us here: https://www.drupal.org/node/2825575. [Paragraphs](https://drupal.org/project/parahraphs) are one of the most widely used Drupal modules. We are happy to spin up a demo environment if that would help in any way.